api/kessel/inventory/v1beta1/policy_detail.proto

```diff
@@ -21,5 +23,5 @@ message PolicyDetail {
   // Defines if the policy is currently enabled or disabled across all targets
   bool disabled = 270940796;
```

Comment:
Note that this is still not correctly modelled. 
Booleans will default to false, but is not possible (as is) to test if they are set or not.

I saw that we might want to default, so it might be fine if that's the case.

Reply:
This should be fine. Disabled = false would be a good default.